src/js/index.js

```diff
@@ -110,6 +110,13 @@ function displayText(json) {
 
   if (prefs['slide-layout']['display-options'].larivaar === true) {
     document.body.classList.add('larivaar');
+    if (prefs['slide-layout']['display-options']['larivaar-assist']) {
```

Comment:
think this needs an else. For older versions of desktop app, the vishraams are not showing when larivaar is on.

Reply:
so logic in css is 
`if NOT larivaar-assist -> show vishraams` 

Even in older versions there should not be a larivaar-assist class.

Reply:
![Image from iOS](https://user-images.githubusercontent.com/1131610/73941856-ff9e6b80-4913-11ea-8a7f-b5836d2d742f.jpg)

Reply:
You right. Forgot to clear my setting after checking out dev